js/removeBook.js

```diff
@@ -0,0 +1,13 @@
+export default function removeBook(e, list) {
+  const newCollectionOfBooks = [];
+
+  if (e.target.matches('.remove')) {
+    const author = e.srcElement.previousElementSibling.textContent;
+
+    const newCollectionOfBooks = list.filter((book) => book.author !== author);
+
+    localStorage.setItem('collection', JSON.stringify(newCollectionOfBooks));
```

Comment:
- [ ] Great job working so far on the functionality for adding a new book but the remove I think is not working properly since when I remove one book all the books are removed so I highly recomend to fix this error to make your project as expected 🔥💪🏻🤩 

### I think it would be great if you made the inputs empty after entering some information that would be great for user experience 🤩💯✨


> hint: you can use fitler on id instead of author since the id it should be unique 🤩💪🏻🔥


https://user-images.githubusercontent.com/58553711/226498362-d04312a8-b6bb-4889-8d19-1917ec16bcbb.mp4

Reply:
Sorry, I made some changes by linters requierements and the code stoped working, now it is working again